### DISCUSSION TITLE
imported zip.js according to the documentation, but get an error when the project runs

### DISCUSSION THREAD

**Discussion by ghost** [gildas-lormeau/zip.js#318] [General]
I imported zip.js according to the documentation, but I get an error when the project runs
```
import * as zip from "@zip.js/zip.js";
```
Here is the error message.

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/57799539/159258716-f0e8b47c-98d0-405d-b9c7-daebc8c4e906.png">

Please what is the cause of the error ？

**Comment by gildas-lormeau** [comment]
Did you already use `import` in your code before integrating zip.js?
